scripts/storage_cleaner.py

```diff
@@ -1440,6 +1471,11 @@ def _add_move_subparser(subparsers: _SubParsersAction):
         action="store_true",
         help="If set, the wandb path for the run is found and appended to the destination dir. If the run is being stored as an archive file, wandb id is first removed from the wandb path and used as the filename.",
     )
+    move_parser.add_argument(
+        "--entry",
+        default=None,
+        help="If provided, moving is restricted to this entry of the run.",
```

Comment:
Maybe add a few words here to describe how an `--entry` should be specified. For example it looks it should be given as a filename without any other path components?

Reply:
https://github.com/allenai/OLMo/pull/494/commits/90648bddb3e2ace234658d60c10c7c1928ba5b11